sdk/identity/identity/src/msal/nodeFlows/msalNodeCommon.ts

```diff
@@ -361,6 +417,21 @@ To work with multiple accounts for the same Client ID and Tenant ID, please prov
       claims: options?.claims,
     };
 
+    if (hasNativeBroker() && this.enableBroker) {
+      if (!silentRequest.tokenQueryParameters) {
+        silentRequest.tokenQueryParameters = {};
+      }
+      if (!this.parentWindowHandle) {
+        // error should have been thrown from within the constructor of Interactive Browser Credential
+        this.logger.warning(
+          "Parent window handle is not specified for the broker. This may cause unexpected behavior.Please provide the parentWindowHandle"
```

Comment:
```suggestion
          "Parent window handle is not specified for the broker. This may cause unexpected behavior. Please provide the parentWindowHandle."
```